crates/stc_ts_file_analyzer/tests/pass/union/any/1.swc-stderr

```diff
@@ -0,0 +1,41 @@
+
+  x Type
+   ,-[$DIR/tests/pass/union/any/1.ts:7:1]
+ 7 | const t1: Any = 1;
+ 8 | t1;
+   : ^^
+   `----
+
+Error: 
+  > any
```

Comment:
This should not be any.

https://www.typescriptlang.org/play?#code/C4TwDgpgBAKlC8UCGA7EUA+UDOwBOAligOaZQDkAZgPbXlnkBGSe5A3FAFDeeUCuKAMbAC1FFD7YIAQTQAKYAC5YASigBvTlG1QClKAoTxEVcis4BfHkA
<img width="755" alt="image" src="https://user-images.githubusercontent.com/29931815/223904867-1efa4249-6125-4418-87ee-cba71325bca6.png">

Reply:
I think the problem here is that https://github.com/dudykr/stc/issues/795 is a wrong issue from start

Reply:
We cannot drop other types even if `any` is a member of a union